dnf/callback.py

```diff
@@ -29,6 +29,8 @@
 PKG_REMOVE = dnf.yum.rpmtrans.TransactionDisplay.PKG_ERASE  # :api
 PKG_UPGRADE = dnf.yum.rpmtrans.TransactionDisplay.PKG_UPGRADE  # :api
 PKG_VERIFY = dnf.yum.rpmtrans.TransactionDisplay.PKG_VERIFY  # :api
+TRANS_PREPARATION = dnf.yum.rpmtrans.TransactionDisplay.TRANS_PREPARATION  # :api
```

Comment:
Other actions use full word for description, but if anyone else would prefere the change, I will change that